common/src/main/java/io/novafoundation/nova/common/view/parallaxCard/ParallaxCardBitmapBaking.kt

```diff
@@ -57,7 +62,7 @@ class ParallaxCardBitmapBaking(private val context: Context, val lruCache: Backi
             try {
                 withContext(Dispatchers.Default) {
                     cardBackgroundBitmap = getBitmapFromCache("cardBackgroundBitmap") {
-                        BitmapFactory.decodeResource(context.resources, R.drawable.ic_parallax_card_background).downscale(0.5f)
+                        BitmapFactory.decodeResource(context.resources, R.drawable.ic_parallax_card_background)
```

Comment:
I guess we dont perform downcaling anymore to be able to use this bitma as placeholder as well. But I dont see us downcasling this bipmap later on before backing burling/processing it?